handbook/product/tracking_issues.md

```diff
@@ -0,0 +1,88 @@
+# Tracking issues
+
+## Table of contents
+
+1. [What is a Tracking issue?](#what-is-a-tracking-issue)
+1. [Creating a Tracking issue](#creating-a-tracking-issue)
+1. [Populating and maintaining a Tracking issue](#populating-and-maintaining-a-tracking-issue)
+1. [Planning a milestone with a Tracking issue](#planning-a-milestone-with-a-tracking-issue)
+1. [Using a Tracking issue for progress check-ins](#using-a-tracking-issue-for-progress-check-ins)
+1. [Contributing to the Tracking issue tool](#contributing-to-the-tracking-issue-tool)
+
+## What is a Tracking issue?
+
+A Tracking issue is a GitHub issue that captures the planned and on-going work of a team's milestone, project, RFC, OKR or anything else of the sort. This artifact is a medium used for planning, progress check-ins and stakeholder communication.
+
+You can take a look at [examples of open tracking issues](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking) to get a sense of what they look like.
+
+## Creating a Tracking issue
+
+1. Create a new GitHub issue with the [Tracking issue template](https://github.com/sourcegraph/sourcegraph/issues/new?assignees=&labels=tracking&template=tracking_issue.md&title=%24TEAM%3A+%24MILESTONE+Tracking+issue).
+1. Give it the **"$THING_BEING_TRACKED: Tracking Issue"** title.
+1. Assign it to the right milestone, creating such milestone if it doesn't yet exist.
+1. Ensure the **tracking** label is set as well as the other labels you want to track. Issues and Pull Requests with those labels will show up in the tracking issue.
+
```

Comment:
````suggestion
1. Add the following lines to the tracking issue where you want all the tracked issues and PRs to be displayed.
   ```
   <!-- BEGIN WORK -->
   <!-- END WORK -->
   ```

````

Reply:
This is contained in the template so probably not necessary to add here?